airflow_bridge_test/requirements.txt

```diff
@@ -1,5 +1,7 @@
 snapshottest==0.6.0
 pytest==3.8.2
+google-cloud-logging==1.11.0
```

Comment:
I don't think you need to add this again, here.

airflow_bridge_test is built from airflow_bridge

Reply:
So the base image should already have these already, due to the changes you made in the Dockerfile, in this PR